src/hotspot/share/opto/superword.cpp

```diff
@@ -1723,7 +1731,14 @@ void SuperWord::construct_my_pack_map() {
     Node_List* p = _packset.at(i);
     for (uint j = 0; j < p->size(); j++) {
       Node* s = p->at(j);
-      assert(my_pack(s) == NULL, "only in one pack");
+#ifdef ASSERT
+      if (my_pack(s) != NULL) {
+        s->dump(1);
+        tty->print_cr("packs[%d]:", i);
+        print_pack(p);
+        assert(false, "only in one pack");
```

Comment:
This is just `fatal`, right?

Reply:
it is assert because this block of code is under `#ifdef ASSERT`